tests/php/unit/Controller/ViewControllerTest.php

```diff
@@ -112,6 +112,7 @@ public function testIndex(): void {
 				['calendar', 'forceEventAlarmType', '', 'forceEventAlarmType'],
 				['dav', 'allow_calendar_link_subscriptions', 'yes', 'no'],
 				['calendar', 'showResources', 'yes', 'yes'],
+				['calendar', 'calendarsOfInterest', ''],
```

Comment:
tests can be fixed by increasing the counter:

`$this->config->expects(self::exactly(16))`

here and in the next method too.